docs/reference/components/aws.cue

```diff
@@ -186,6 +186,12 @@ components: _aws: {
 						account access.
 						"""
 				},
+				{
+					title: "Expiration"
+					body: """
+						Vector will again search for AWS credentials once they expire.
```

Comment:
I might just put this in the top "AWS Authentication" section. If they expire, it checks all credential mechanisms, yes?

Reply:
Yes, it rechecks as if it's fetching them for the first time.